CefSharp/DefaultRequestHandler.cs

```diff
@@ -0,0 +1,96 @@
+﻿// Copyright © 2010-2017 The CefSharp Authors. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style license that can be found in the LICENSE file.
+
+using System.Security.Cryptography.X509Certificates;
+
+namespace CefSharp
```

Comment:
At a minimum this should be in the `Handler` folder. Probably the `CefSharp.Handler` namespace as well. (Ideally all the handlers should be in their own namespace, moving the interfaces is a massive breaking change, so probably won't happen).